src/annotate.ts

```diff
@@ -7,10 +7,21 @@ import * as fs from 'fs';
 const schema: any = JSON.parse(fs.readFileSync('./schemas/annotate-is-valid.json', 'utf8'));
 const ajv = new Ajv({ allErrors: true, verbose: true });
 
-export default async function annotate({payload, _ajv = ajv, _schema = schema}: { payload: { field: string, term: string }, _ajv: typeof ajv, _schema: typeof schema}) {
+function confidenceScoreF(string: string) {
```

Comment:
Could you add a comment explaining what you try to do with this function?
Do not use the word `string` as the name of argument, it clashes with the reserved word for the type, use something like `str` instead.